packages/opentelemetry-metrics/src/types.ts

```diff
@@ -28,9 +28,6 @@ export interface MetricOptions {
   /** The unit of the Metric values. */
   unit: string;
 
-  /** The list of label keys for the Metric. */
-  labelKeys: string[];
```

Comment:
This is still needed. See [specs](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/api-metrics-user.md#recommended-label-keys). I'd suggest to keep it here.